elf2flt.c

```diff
@@ -977,45 +999,20 @@ output_relocs (
 
 #ifdef TARGET_microblaze
 				case R_MICROBLAZE_64:
-		/* The symbol is split over two consecutive instructions.  
-		   Flag this to the flat loader by setting the high bit of 
-		   the relocation symbol. */
-				{
-					unsigned char *p = r_mem;
-					pflags=0x80000000;
-
 					/* work out the relocation */
 					sym_vma = bfd_section_vma(abs_bfd, sym_section);
 					sym_addr += sym_vma + q->addend;
 					/* Write relocated pointer back */
-					p[2] = (sym_addr >> 24) & 0xff;
-					p[3] = (sym_addr >> 16) & 0xff;
-					p[6] = (sym_addr >>  8) & 0xff;
-					p[7] =  sym_addr        & 0xff;
-
-					/* create a new reloc entry */
-					flat_relocs = realloc(flat_relocs,
-						(flat_reloc_count + 1) * sizeof(uint32_t));
-					flat_relocs[flat_reloc_count] = pflags | (section_vma + q->address);
-					flat_reloc_count++;
-					relocation_needed = 0;
-					pflags = 0;
-			sprintf(&addstr[0], "+0x%ld", sym_addr - (*(q->sym_ptr_ptr))->value -
-					 bfd_section_vma(abs_bfd, sym_section));
-			if (verbose)
-				printf("  RELOC[%d]: offset=0x%"BFD_VMA_FMT"x symbol=%s%s "
-					"section=%s size=%d "
-					"fixup=0x%x (reloc=0x%"BFD_VMA_FMT"x)\n",
-					flat_reloc_count,
-					q->address, sym_name, addstr,
-					section_name, sym_reloc_size,
-					sym_addr, section_vma + q->address);
-			if (verbose)
-				printf("reloc[%d] = 0x%"BFD_VMA_FMT"x\n",
-					 flat_reloc_count, section_vma + q->address);
-
-					continue;
-				}
+					r_mem[2] = (sym_addr >> 24) & 0xff;
+					r_mem[3] = (sym_addr >> 16) & 0xff;
+					r_mem[6] = (sym_addr >>  8) & 0xff;
+					r_mem[7] =  sym_addr        & 0xff;
+					relocation_needed = 1;
+					/* The symbol is split over two consecutive instructions.  
+					   Flag this to the flat loader by setting the high bit of 
+					   the relocation symbol. */
+					pflags=0x80000000;
```

Comment:
i know you're just moving code around, but please add spaces around the `=` while you're here